schema/functions.json

```diff
@@ -90,6 +90,25 @@
         "name",
         "operation"
       ]
+    },
```

Comment:
i dont think this is right. we should find a way to restrict object type on the value of type property please.

Reply:
also description if object type should add something like "added only for rest type"

Reply:
I don't think so. The `AuthRef` object can be applied to any remote invocation such as gRPC, GraphQL, openapi, rest, odata, etc.

If we would make this change, I'd say to do it in another PR. There's no change here apart from the formatting and the default to `openapi` type.

Reply:
Resolved?